templates/layouts/_base.html

```diff
@@ -139,3 +140,25 @@
     //]]></script>
   {% endif %}
 {% endblock %}
+
+{% block preFooter %}
+  {% if session_expires_at %}
+    {{
+      onsTimeoutModal({
+        "showModalTimeInSeconds": 60,
+        "serverSessionExpiryEndpoint": url_for('session.session_expiry'),
+        "sessionExpiresAt": session_expires_at,
+        "redirectUrl": url_for('session.get_signed_out'),
```

Comment:
Need to confirm with Donna/Sarah, but I think the content on this page needs updating. This didn't match the content you get on a 401.html, which includes the bullet point about being timed out etc.

Reply:
Checked with Donna and it's the text we've been given, so no changes.

Reply:
Have clarified this should go to `session.get_session_expired` which has the right content since it more that it has expired vs they signed out.

Reply:
Ok, refactored the test to reflect the changes, thanks!